apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -290,6 +316,7 @@ const BookingPage = ({
         eventTypeSlug: eventType.slug,
         seatReferenceUid: "seatReferenceUid" in responseData ? responseData.seatReferenceUid : null,
         ...(rescheduleUid && booking?.startTime && { formerTime: booking.startTime.toString() }),
+        ...getBookingRedirectExtraParams(responseData),
```

Comment:
This should be done as part of `bookingSuccessRedirect` and that fn is called at multiple places as redirections happens from multiple places.